BenchmarkCNV/BenchmarkCNV.wdl

```diff
@@ -4,114 +4,199 @@ workflow Benchmark_CNV_Caller {
+        File eval_cnv_vcf
+        File wittyer_cnv_config
+        String wittyer_cnv_evaluation_mode
+        File eval_sv_vcf
+        File wittyer_sv_config
+        String wittyer_sv_evaluation_mode
+        String wittyer4mat_docker
     }
 
     # Select vcf for specific sample
     call SelectVariant {
         input:
-        gatk_docker = gatk_docker,
-        vcf = variant_callset,
-        sample_name = sample_name
+            gatk_docker = gatk_docker,
+            vcf = variant_callset,
+            truth_sample_name = truth_sample_name
     }
 
-    # benchmark cnv.vcf using witty.er
+    # benchmark cnv.vcf and sv.vcf using witty.er tool
     call BenchmarkCNV {
         input:
-        wittyer_docker = wittyer_docker,
-        truth_vcf = SelectVariant.output_vcf,
-        eval_vcf = eval_vcf,
-        sample_name = sample_name,
-        config_file = wittyer_config,
-        evaluation_mode = wittyer_evaluation_mode
+            wittyer_docker = wittyer_docker,
+            truth_vcf = SelectVariant.output_vcf,
+            truth_sample_name = truth_sample_name,
+            query_sample_name = query_sample_name,
+            eval_cnv_vcf = eval_cnv_vcf,
+            cnv_config_file = wittyer_cnv_config,
+            cnv_evaluation_mode = wittyer_cnv_evaluation_mode,
+            eval_sv_vcf = eval_sv_vcf,
+            sv_config_file = wittyer_sv_config,
+            sv_evaluation_mode = wittyer_sv_evaluation_mode
+    }
+
+    # wittyer4mat to parse the wittyer json output
+    call Wittyer4Mat {
+        input:
+            wittyer4mat_docker = wittyer4mat_docker,
+            cnv_wittyer_stats = BenchmarkCNV.cnv_wittyer_stats,
+            sv_wittyer_stats = BenchmarkCNV.sv_wittyer_stats,
+            truth_sample_name = truth_sample_name
     }
 
     # Outputs that will be retained when execution is complete
     output {
         File truth_vcf = SelectVariant.output_vcf
-        File wittyer_annotated_vcf = BenchmarkCNV.wittyer_annotated_vcf
-        File wittyer_stats = BenchmarkCNV.wittyer_stats
+        File cnv_wittyer_stats = BenchmarkCNV.cnv_wittyer_stats
+        File cnv_wittyer_annotated_vcf = BenchmarkCNV.cnv_wittyer_annotated_vcf
+        File cnv_wittyer_annotated_vcf_index = BenchmarkCNV.cnv_wittyer_annotated_vcf_index
+        File sv_wittyer_stats = BenchmarkCNV.sv_wittyer_stats
+        File sv_wittyer_annotated_vcf = BenchmarkCNV.sv_wittyer_annotated_vcf
+        File sv_wittyer_annotated_vcf_index = BenchmarkCNV.sv_wittyer_annotated_vcf_index
+        File cnv_deletion_stat = Wittyer4Mat.cnv_deletion_stat
+        File cnv_duplication_stat = Wittyer4Mat.cnv_duplication_stat
+        File sv_deletion_stat = Wittyer4Mat.sv_deletion_stat
+        File sv_duplication_stat = Wittyer4Mat.sv_duplication_stat
+        File sv_insertion_stat = Wittyer4Mat.sv_insertion_stat
     }
 }
 
     # Task 1: Select sample vcf from a large callset (e.g. 1KGP)
     task SelectVariant {
```

Comment:
I think it would be better to rename this from SelectVariant to SelectSample or something like that

Reply:
will do it in the next commit.